### PR TITLE
Update readme.md and libraries.md

### PR DESCRIPTION
**Pull request by AlejandraPedroza**
This PR contains updates to the readme.md and libraries.md docs.

### CODE REVIEW

**Review by ileasile — CHANGES_REQUESTED** [review]
I like text improvements, but also have some comments

File: docs/README-STUB.md [hunk 1]
```diff
@@ -7,437 +7,699 @@
 [![GitHub](https://img.shields.io/github/license/Kotlin/kotlin-jupyter)](https://www.apache.org/licenses/LICENSE-2.0)
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/kotlin/kotlin-jupyter/master?filepath=samples)
 
-# Kotlin Kernel for IPython/Jupyter
+# Kotlin Kernel for Jupyter notebooks
 
-[Kotlin](https://kotlinlang.org/) ([[kotlin_version]]) [kernel](https://docs.jupyter.org/en/latest/projects/kernels.html) for [Jupyter](https://jupyter.org).
+The Kotlin Kernel for Jupyter notebooks is a powerful tool that allows you to write and run [Kotlin](https://kotlinlang.org/) code within the
+[Jupyter Notebook](https://jupyter.org) environment. This [Kernel](https://docs.jupyter.org/en/latest/projects/kernels.html) essentially acts as a bridge between Jupyter Notebook and the Kotlin compiler.
```
[the remark this answers is not in the document]
Comment (AlejandraPedroza) [reply docs/README-STUB.md:13]: I'm leaving capitalized when we refer specifically to Kotlin Kernel as per our discussion with MKT.

File: docs/README-STUB.md [hunk 2]
```diff
@@ -7,437 +7,699 @@
...
+## Install the Kotlin Kernel for notebooks
+
+You can use the Kotlin Kernel for notebooks on:
+* [IntelliJ IDEA](https://www.jetbrains.com/idea/)
+* [Jupyter Notebook](https://jupyter.org/)
+* [Datalore](https://www.jetbrains.com/datalore/)
 
-<!-- Start Document Outline -->
+To use our Kernel in IntelliJ IDEA, you need to install the [Kotlin Notebook plugin](#install-the-kotlin-notebook-plugin). To use it in your Jupyter notebooks,
+you can install it through [Conda](#install-with-conda), [Pip](#install-with-pip), or [sources](#install-from-sources).
```
[the remark this answers is not in the document]
Comment (AlejandraPedroza) [reply docs/README-STUB.md:120]: What about now?

**Comment by ileasile** [comment]
I merged all of your changes, thank you:
https://github.com/Kotlin/kotlin-jupyter/commit/9427cc613c6124194c9a335ede24d710c8cf9c22

You can verify README in master to check that everything looks like you expected

**Pull request closed without merging**